wbb/modules/karma.py

```diff
@@ -129,7 +129,7 @@ async def downvote(_, message):
 
 @app.on_message(filters.command("karma") & filters.group)
 @capture_err
-async def karma(_, message):
+async def command_karma(_, message):
     chat_id = message.chat.id
     if not message.reply_to_message:
         m = await message.reply_text("Analyzing Karma...Will Take 10 Seconds")
```

Comment:
just for the record, changed the function name because "karma" is a shadow name. (there is a same name variable is being used inside the function).